splunk_add_on_ucc_framework/commands/openapi_generator/ucc_object.py

```diff
@@ -0,0 +1,36 @@
+from pathlib import Path
+from typing import List
+from splunk_add_on_ucc_framework.commands.openapi_generator.json_to_object import DataClasses
+
+class GlobalConfig(DataClasses):
+    def _validate_actions_list(self, *, actions: List[str]) -> None:
+        ALLOWED_VALUES = {"edit", "delete", "clone"}
+        for action in actions:
+            if action not in ALLOWED_VALUES:
+                raise ValueError(f"Action {action} not in the list of allowed values ({ALLOWED_VALUES})")
```

Comment:
I think this needs to be validated during the globalConfig validation, not here.

Reply:
This one is deleted in the new PR, we are already checking for those values in the globalConfig schema.